intro changes for smooth migration zeebe user tasks next (#4723)
camunda/camunda-docs: docs/apis-tools/zeebe-api-rest/tutorial.md

:::note

In this tutorial, we will execute arguments to assign and unassign a user to and from a Camunda user task. You can examine the framework for processing these arguments in the `cli.js` file before getting started.

:::

## Assign a Camunda user task (POST)

:::note

> In this tutorial, you will capture a **Camunda user task** ID to assign and unassign users in this API. Camunda 8.5 introduced this new [user task](/components/modeler/bpmn/user-tasks/user-tasks.md) implementation type, and these Camunda user tasks are different from job worker-based user tasks. See more details on task type differences in the [migrating to Camunda user tasks documentation](/apis-tools/migration-manuals/migrate-to-camunda-user-tasks.md#task-type-differences).

Reviewer: ```suggestion
In this tutorial, you will capture a **Camunda user task** ID to assign and unassign users in this API. Camunda 8.5 introduced this new [user task](/components/modeler/bpmn/user-tasks/user-tasks.md) implementation type, and these Camunda user tasks are different from job worker-based user tasks. To learn m ore about task type differences, see [migrating to Camunda user tasks](/apis-tools/migration-manuals/migrate-to-camunda-user-tasks.md#task-type-differences).
```

`const zeebeApiUrl = process.env.ZEEBE_BASE_URL`

> 5. On the next line, script the API endpoint to assign a Camunda user task.:

Reviewer: ```suggestion
5. On the next line, script the API endpoint to assign a Camunda user task:
```
